### DISCUSSION TITLE
Migration Reset

### DISCUSSION THREAD

**Discussion by rodolphonetto** [adonisjs/core#1640] [Help]
Hi, I'm trying to run

`node ace migration:reset`

✖  error     "migration:reset" command not found

the command change from adonis v4.1? I did not found anything about reset on the docs

thanks for the time :)

**Comment by thetutlage** [comment]
Yup. Not added to v5 yet.

**Comment by fnoquiq** [comment]
Try it:

Run multiple times until reset: `node ace migration:rollback`.

And then just run it once: `node ace migration:run`.

**Reply by thetutlage** [reply]
Nope, there is even a better way. `node ace migration:rollback --batch=0`
